### PR TITLE
Create ms.json

### PR DESCRIPTION
**Pull request by samanthawxlow**
A Bahasa Malaysian translation for Find a Doc

### CODE REVIEW

File: my.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "title": "Senarai Menunggu Pembatalan Vaksinasi COVID-19 di Jepun"
+    },
+    "login": {
+        "clear": "Kosongkan",
+        "email": "E-mel",
+        "login": "Daftar masuk",
+        "password": "Kata laluan"
+    }
+}
+© 2021 GitHub, Inc.
```
Comment (ann-kilzer) [inline my.json:50]: Can we remove these lines? It looks like they were accidentally copied in.

If you need help I can do it for you!

**Review by ann-kilzer — COMMENTED** [review]
This is awesome!

Let's call this file `ms.json` because we are using the [two-letter language codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) rather than the country codes. (And I'll make that clearer on the README)

And it looks like a few extra lines got copied in?

File: my.json (hunk 1 above)
Comment (samanthawxlow) [reply my.json]: Done! Thank you. Have renamed to  ms.json and removed the weird lines.

**Review by ann-kilzer — APPROVED** [review]
LGTM 🧇

**Pull request merged**